conf/route_control.py

```diff
@@ -275,7 +276,7 @@ def delete_module(self, module_name: str) -> None:
         """Deletes a BESS module.
 
         Args:
-            update_module (str): The name of the module to delete.
+            module_name (str): The name of the module to delete.
```

Comment:
Thank you for those docstring changes, it seems a bunch of them did not match the actual input parameters